src/bin/packageJson.test.ts

```diff
@@ -0,0 +1,15 @@
+import { describe, expect, it } from "vitest";
+
+import { getVersionFromPackageJson } from "./packageJson.js";
+
+describe("Get version number from package.json", () => {
+	it("Returns a string representing a version number when given a valid package.json URL", async () => {
+		const version = await getVersionFromPackageJson();
+
+		expect(version).toBeTypeOf("string");
+
+		if (typeof version === "string") {
```

Comment:
[Cleanup] https://github.com/veritem/eslint-plugin-vitest/blob/main/docs/rules/no-conditional-expect.md - I'm surprised it's not triggering here. Maybe a bug to file somewhere?

If the `if` is just to make sure `version` is a string, you can use an `as` assertion instead.

That being said, I think you can actually use `import` a little more freely in tests. Vitest massages some runtime stuff nicely for us so we don't have to worry about path relativity or whatnot.

```ts
import packageData from "../../package.json";
```

```ts
expect(version).toBe(packageData.version);
```

Reply:
I've removed it. Now that I throw an error if its not a string it would be irrelevant anyway.